dbaas/tsuru/views.py

```diff
@@ -576,6 +484,17 @@ def check_database_status(database_name, env):
         return log_and_response(
             msg=msg, http_status=status.HTTP_412_PRECONDITION_FAILED)
 
+    task = TaskHistory.objects.filter(
+        arguments__contains="Database: {}, Environment: {}".format(
+            database_name, env
+        ), task_status="ERROR")
+
```

Comment:
This filter will get any task for this database when status = ERROR

Reply:
U right. We will back later to refactor all legacy method ok ?